🎟 Wrap event fixtures the way they're supposed to be (#45)
IBM/slack-wrench: packages/jest-bolt-receiver/README.md

### Sending Events

> We recommend you use [@slack-wrench/fixtures](../fixtures) to generate and send events. That way, you can easily generate fixtures and have as global overrides and customizing payloads. Check its documentation for details.

Reviewer: ```suggestion
We recommend you use [@slack-wrench/fixtures](../fixtures) to generate and send events. That way, you can easily generate fixtures, set global overrides, and customize payloads. Check its documentation for details.
```
